commands/TenorCommand.ts

```diff
@@ -48,6 +48,8 @@ export class TenorCommand implements ISlashCommand {
         try {
             const image = await this.app.getImageGetter().getOne(this.app.getLogger(), http, item.id, read);
             const showTitle = await read.getEnvironmentReader().getSettings().getValueById('tenor_show_title');
+            const showLink = (await read.getEnvironmentReader().getSettings().getById('tenor_show_link')).value;
```

Comment:
Thanks for your contribution! Is there a reason why you're getting this value differently than the one right above it?

Reply:
Yeah, good question. For some reason `getValueById` was just always `true`. I have no clue as to why. I changed it to `getById` to get some more information and see if the value was actually changed in the database and it just started working. I really don't know why.